src/ray/raylet/node_manager.cc

```diff
@@ -651,36 +666,28 @@ void NodeManager::ProcessDisconnectClientMessage(
       // If the worker was killed intentionally, e.g., when the driver that created
       // the task that this worker is currently executing exits, the task for this
       // worker has already been removed from queue, so the following are skipped.
-      auto const &running_tasks = local_queues_.GetRunningTasks();
```

Comment:
curious, why do you not need to do this anymore?

Reply:
was it just redundant before?

Reply:
It was only used to get the task from the taskID in order to get the driver ID, but now there is a more efficient way, e.g., just by calling 

```cpp
const JobID &job_id = worker->GetAssignedDriverId();
```

also, it was buggy because the task might have been in the *blocked queue* instead of the *running queue*.